django/thunderstore/repository/templates/repository/includes/dependencies.html

```diff
@@ -1,22 +1,28 @@
 {% load thumbnail %}
 {% load community_url %}
 
-<div class="list-group">
-    <div class="list-group-item flex-column align-items-start active">
+<div class="card">
+    <div class="card-header bg-primary pb-2">
         <h4>This mod requires the following mods to function</h4>
     </div>
-    {% for dependency in object.package.dependencies.all %}
-    <div class="list-group-item flex-column align-items-start media">
-        <div class="media">
-            <img class="align-self-center mr-3" src="{% thumbnail dependency.icon 64x64 %}" alt="{{ dependency }} icon">
-            <div class="media-body">
-                <h5 class="mt-0"><a href="{% community_url "packages.detail" owner=dependency.package.namespace.name name=dependency.package.name %}">
-                    {{ dependency.package }}
-                </a></h5>
-                <p class="mb-0">{{ dependency.description }}</p>
-                <small class="mb-0">Preferred version: <a href="{% community_url "packages.version.detail" owner=dependency.package.namespace.name name=dependency.package.name version=dependency.version_number %}">{{ dependency.version_number }}</a></small>
-            </div>
+    <div class="card-body p-0">
+        <div class="list-group" style="max-height:450px; overflow-y:scroll;">
```

Comment:
This will now always include a scrollbar even when one isn't needed. Also it would be more appropriate to use a custom css class rather than inlining.